src/aspire/utils/optimize.py

```diff
@@ -29,7 +29,7 @@ def fill_struct(obj=None, att_vals=None, overwrite=False):
         return obj
 
     for key in att_vals.keys():
-        if hasattr(obj, key):
+        if key in obj.keys():
             if overwrite:
```

Comment:
What is the use case for `overwrite`?

Reply:
Set the option whether the key value in the obj can be overwrite or not by that of att_vals.

Reply:
remove the `overwrite` option.